### PR TITLE
Issue 1 set goal weight

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-04-07 at 9 42 58 PM

### CODE REVIEW

File: client/components/form.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        <label>Current Weight</label>
+        <input
+        required
+        type="number"
+        onChange={this.handleChange}
+        min="100"
+        name="currentWeight"
+        placeholder="Pounds">
+        </input>
+        </div>
```
Comment (thebearingedge) [inline client/components/form.jsx]: This indentation is very hard to follow. Confirm if you linter is working, but child elements should always be indented within their parent.

File: server/public/styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: adding `* { box-sizing: border-box; }` as well makes working with padding/margin _much_ easier.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, good job so far, your code just needs some better formatting. Also, here are a couple of visual differences from your original design that would be good to fix:

- your input borders are not the right color
- your inputs (and select) are not the right height 
- input labels do not have enough bottom margin
- input border radius looks off
- page header is not tall enough/text is not vertically centered (this is usually fixed with padding)

File: server/public/styles.css (hunk 2 above)
Comment (yesseniarod) [reply server/public/styles.css]: Screen Shot 2021-04-08 at 11 46 11 AM

**Review by thebearingedge — APPROVED** [review]
